### DISCUSSION TITLE
Backing up the KeePassXC database to a different location

### DISCUSSION THREAD

**Discussion by ScottinCanada** [keepassxreboot/keepassxc#6114] [Q&A] [answered]
Basic backup principles say it's wise to always backup your data to a totally different location. Quoting from the KeePassXC manual...  "It is a good practice to create copies of your database file and store the copies of your database on a different computer, smart phone, or cloud storage space such a Google Drive or Microsoft OneDrive." No argument there.

After turning on the "Backup database file before saving" option in the General Settings area, all that does is create another copy of the database with "old" appended to the database name, in the same location/folder where the main database is. It does not offer an option to specify a different target location. Not much help there. Of course there is the manual approach of clicking Database on the toolbar, followed by "Save Database Backup...", and specify a different location and name for the backup file. That gets the job done but I'd better remember to do that from time to time.

Am I missing something, or does that about sum it up? In other words, KeePassXC  alone doesn't do a complete backup job. I would need to make sure to include the main database file with other critical files while doing other backups in general.

Thanks for all comments!

**Comment by phoerious** [comment, marked as answer]
We expect that you are using an external backup solution. This feature is simply a failsafe in case something goes wrong while saving your database.

**Reply by ScottinCanada** [reply]
OK thanks for the reply. You confirmed what I was thinking.
